### ISSUE TITLE
Make bundling work with custom output location

### ISSUE THREAD

**Issue by janpio** [prisma/prisma#12607]
It seems currently bundling can not work properly for the generated Prisma Client when it is generated to a custom `output` location. Problem: No `package.json` generated in custom output path because of `if` in code, means no information for Bundler to know what file(s) to include.

Next steps:
* Investigate if webpack can pickup package.json files in custom locations (out of node_modules)

**Comment by millsp** [comment]
(cc @Frankitch @VincentThomas06)

**Comment by Frankitch** [comment]
Hi @millsp  @VincentThomas06, I opened an issue a couple of weeks ago (#12296) which has a similar problem but browser side. This issue can be solved by adding a `package.json` file. Knowing that I have already tried to add a `package.json` file to solve this current issue but I did not succeed, I hope I did something wrong...

**Comment by joshribakoff-sm** [comment]
I have a project with multiple Prisma schemas, there is a default one and then one that outputs to `./node_modules/@internal/`.

We are constantly getting this error, happens around 50% of the time. Re-running our script to run all of the code generators fixes it. It seems like some sort of race condition.

```
ERROR in ./node_modules/@internal/prisma/client/runtime/index.js 27652:17-40
Module not found: Error: Can't resolve '_http_common' in '/Users/jribakoff/groundwater/node_modules/@internal/prisma/client/runtime'
```

> Investigate if webpack can pickup package.json files in custom locations (out of node_modules)

I would like to respectfully question if ya'll pinned down the exact issue, because I am not outputting anything outside of `node_modules`, furthermore the fact it works sometimes and not others does not seem to jive with your understanding of the issue

**Comment by SevInf** [comment]
@joshribakoff-sm what versions of `prisma` and `@prisma/client` are you using? `_http_common` issue was solved in `3.12 ` and as of `4.1.0` I can't see this import in prisma runtime code. If you are using older version, please, update. If you see this issue on a current version I'd kindly ask you to submit a reproduction.

**Comment by JonParton** [comment]
Just a little comment to say we have also encountered this issue where we want to generate our client to a known location rather then the automatic prisma behaviour. This is driven by us needing to use two prisma clients in our project for different databases. 

By outputting to a folder outside `node_modules` in our next.js app it caused a `Can't resolve 'util/types' error` in the build as has been mentioned in these issues. 

For now I have resolved using [this workaround](https://github.com/prisma/prisma/issues/6899#issuecomment-1057748764) and published to two known locations within `node_modules` that we define but I know in future this won't play nice with mono repo's and pnpm etc 😔
